packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -126,7 +126,7 @@ const groupByControl: SharedControlConfig<'SelectControl', ColumnMeta> = {
     if (state.datasource) {
       const options = state.datasource.columns.filter(c => c.groupby);
       if (includeTime) {
-        options.push(timeColumnOption);
+        options.unshift(timeColumnOption);
```

Comment:
Move time column to the first option for `groupby`. It's hard to notice this is an auto-generated column when it's at the end of the list.